src/test/java/com/pi4j/crowpi/components/SoundSensorComponentTest.java

```diff
@@ -0,0 +1,61 @@
+package com.pi4j.crowpi.components;
+
+import com.pi4j.crowpi.ComponentTest;
+import com.pi4j.io.gpio.digital.DigitalInput;
+import com.pi4j.io.gpio.digital.DigitalState;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class SoundSensorComponentTest extends ComponentTest {
+    protected SoundSensorComponent soundSensor;
+    protected DigitalInput din;
```

Comment:
We can make use of the newly introduced mock helpers which I added as part of the RFID component, so you can use the type `MockDigitalInput` here instead.